data/presets/fields/flats.json

```diff
@@ -0,0 +1,7 @@
+{
+    "key": "flats",
+    "type": "number",
+    "minValue": 0,
+    "label": "Flats",
```

Comment:
So iD's default language is American English. Could we make the label "Units" instead?